app/src/androidTest/java/com/github/HumanLearning2021/HumanLearningApp/firestore/FirestoreDatabaseServiceTest.kt

```diff
@@ -312,4 +321,15 @@ class FirestoreDatabaseServiceTest {
     fun test_getRepresentativePicture() = runBlocking {
         assertThat(demoInterface.getRepresentativePicture(appleCategoryId), not(equalTo(null)))
     }
+
+    @Test
+    fun test_setAdminAccess() {
+        runBlocking {
+            val firebaseUser = Firebase.auth.signInAnonymously().await().user!!
+            presenter.onSuccessfulLogin(false)
+            val fireStoreUser = scratchInterface.setAdminAccess(firebaseUser, true)
```

Comment:
```suggestion
            val user = scratchInterface.setAdminAccess(firebaseUser, true)
```
- having two fire[blah]User is confusing
- The `User` interface is a thing
- The `FirestoreUser` is going away